ImprovedAnvil/scripts/encounters/graveyard/S!crypt.baf

```diff
@@ -8,6 +8,14 @@ THEN
     SetGlobal("Iaqueen","AR0806",1)
 END
 
+IF

```

Comment:
use two spaces instead of tabulation here, please (you can probably set this up in the text editor as the default behavior)
also, maybe have this thing dropped only when Hexxat is in party?

Reply:
prefer not to have conditional drops